### DISCUSSION TITLE
I am unable to run first command

### DISCUSSION THREAD

**Discussion by Shahruhban01** [termux/termux-app#3057] [Q&A] [answered]
Please help me....i am unable to run apt update and apt upgrade commands

**Comment by roseaguilar1979** [comment]
apt update && apt upgrade

**Comment by sylirre** [comment, marked as answer]
Don't use Termux from Play Store. It is dead like zombie. Stay away from Play Store installation source. Uninstall it immediately.

Use F-Droid builds instead. For example, v0.118 is current latest version of Termux and you can get it here: https://f-droid.org/repo/com.termux_118.apk

After installation, run this command:
```
yes | pkg upgrade
```

**Comment by pressfanksss** [comment]
> unable

and what's the problem?
